lib/modules/platform/azure/index.ts

```diff
@@ -64,7 +65,7 @@ interface Config {
   owner: string;
   repoId: string;
   project: string;
-  prList: AzurePr[];
+  prList?: AzurePr[];
```

Comment:
revert and check, otherwise we get into runtime trouble

Reply:
okey

Reply:
how do i invalidate then @viceice ? 
```
  if (!config.prList || !pr || !idx) {
    // if pr doesn't exist in cache then invalidate cache so we get new list from api next time.
    config.prList = undefined;
    return;
  }
  ```

Reply:
i thought if i set it for empty array, and i check length later on.. it would work. but i dont think  that works since empty array means there are no PRs on the repo and its valid

Reply:
ill do config.prList ??= [];

Reply:
config.prList = undefined as any;
?? xD